### ISSUE TITLE
False positive with no-inline-comments in ESLint v4

### ISSUE THREAD

**Issue by soda0289** [eslint/typescript-eslint-parser#215]
**What version of TypeScript are you using?**
2.2.1

**What version of `typescript-eslint-parser` are you using?**
2.1

**What code were you trying to parse?**

```ts
const a = `${folder}/**/*.ts`;

```

**What did you expect to happen?**
It should not error for rule no-inline-comments

**What happened?**
It showed error for no-inline-comments. When using espree parser the error goes away.

**Comment by soda0289** [comment]
Duplicate of #216
